docs/onboarding/installation.md

```diff
@@ -65,20 +50,36 @@ docker run --rm --tty --interactive --network host --volume <home-directory>/.ir
 
 **Note:** You can also use the RPC layer over TCP by starting your node with `--rpc.tcp`, which connects over the default port 8020.
 
-To run a miner replace `<threads>` with a number of threads to use for workers.
+#### Updating
+
+To update your image, use `docker pull` before running the image.
 
 ```sh
-docker run --rm --tty --interactive --network host --volume <home-directory>/.ironfish:/root/.ironfish ghcr.io/iron-fish/ironfish:latest miners:start --threads=<threads>
+docker pull ghcr.io/iron-fish/ironfish:latest
 ```
 
-### Updating
+### From source
 
-To update your image, use `docker pull` before running the image.
+First, head over to GitHub to clone our [official repository](https://github.com/iron-fish/ironfish). Once that's done, follow the initial setup steps in the [README](https://github.com/iron-fish/ironfish#initial-setup) to install prerequisites, then head to the [CLI README](https://github.com/iron-fish/ironfish/tree/master/ironfish-cli#use-scenarios) to run the CLI.
+
+### Homebrew (macOS)
+
+> **Note:** Our Homebrew tap does not yet support **M1/arm64**. However, you can install Iron Fish directly [from the source](#from-source) or [use Docker](#using-docker) with x86_64 support.
```

Comment:
Minor, but should x86_64 be wrapped with backticks?

Reply:
It probably should!